### PR TITLE
Issue 484: absolute taxonomic abundance analysis

### PR DESCRIPTION
**Pull request by johnwjudkins**
Resolves issue #484.

### CODE REVIEW

File: src/ontology/modules/eupath_terms.owl [hunk 1]
```diff
@@ -4775,6 +4775,111 @@ CD3T188G is a polymophism on the human CD36 gene.</obo:IAO_0000116>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/EUPATH_0009350 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/EUPATH_0009350">
+        <obo:IAO_0000115 xml:lang="en">A data transformation of input sequence data to obtain the fraction of absolute abundance of organisms.</obo:IAO_0000115>
```
Comment (asizemore) [inline src/ontology/modules/eupath_terms.owl:4781]: Can we remove "fraction of" in this sentence? The data is always a count of something, not a fraction. So it'd read "...to obtain the absolute abundance...".

**Review by asizemore — COMMENTED** [review]
Just one question - otherwise looks great!

File: src/ontology/modules/eupath_terms.owl (hunk 1 above)
Comment (johnwjudkins) [reply src/ontology/modules/eupath_terms.owl:4781]: @asizemore So "absolute" here means a count of organisms of some type, while "relative" is a fraction? Just making sure that's correct.

**Review by asizemore — APPROVED** [review]
Wonderful! Thanks so much @johnwjudkins !

**Pull request merged**